lambda-function/src/app.ts

```diff
@@ -62,7 +63,9 @@ export const lambdaHandler = async (event: S3Event): Promise<Measure> => {
       logAndMail(`Measure name: ${matMeasure?.manageMeasureDetailModel.measureName}`);
       logAndMail("Measure version in MADiE: 0.0.000");
       logAndMail(`Measure cqlLibraryName: ${matMeasure?.manageMeasureDetailModel.cqllibraryName}`);
-      logAndMail(`CMS ID: ${matMeasure?.manageMeasureDetailModel.eMeasureId}`);
+      if (cmsId) {
+        logAndMail(`CMS ID: ${cmsId}`);
```

Comment:
Can we add an extra test to bring the PR coverage up for when the cmsId is non-zero? (Coverage is reporting this line as unhit)

Reply:
should be added now.